packages/cozy-konnector-libs/docs/synchronization.md

```diff
@@ -67,10 +67,16 @@ To store synchronization data, we are using a `sync` attribute in document `meta
 {
   "metadata": {
     "sync": {
-      "id": "7ee401e841c94159addb47f190903139",
-      "konnector": "trainline",
-      "last_sync": "Mon, 12 Feb 2018 16:25:34 GMT"
-    }
+      "7ee401e841c94159addb47f190903139": {
+        "id": "7ee401e841c94159addb47f190903139",
+        "konnector": "trainline",
+        "last_sync": "Mon, 12 Feb 2018 16:25:34 GMT"
```

Comment:
Why isn't this a ISO date ?

Reply:
As the co-writer of this spec, tell me! :D 

Being serious, you'll use the ISO format `2018-10-19T10:58:37.025688+02:00` if @gregorylegarec has no objection ;)